### PR TITLE
QE-2930: Run Github Action

### PR DESCRIPTION
**Pull request by spcaipers-arm**
Outside this commit, it was done:
* Configured AVH on AWS Account
* Created IoT Thing
* Created GH Secrets

This commit enables GitHub Actions to run this project in the ARM AWS account It installs CMSIS-Toolbox v1.2.0 in the AVH instance

### CODE REVIEW

**Comment by spcaipers-arm** [comment]
Hi @RobertRostohar, this PR will failed because it was created from a fork repo, so the secrets are not accessible.

It is working in my fork:
https://github.com/spcaipers-arm/AWS_MQTT_MutualAuth_SW_Framework/actions/runs/3273912386/jobs/5386740130

I already created the Secrets for this repo.
Once this PR is merged, we can test it. It should work out of the box.

File: .github/workflows/virtual_hardware.yml [hunk 1]
```diff
@@ -71,7 +73,7 @@ jobs:
       uses: aws-actions/configure-aws-credentials@v1
       with:
         role-to-assume: ${{ secrets.AWS_ASSUME_ROLE }}
-        aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+        aws-region: ${{ env.AWS_DEFAULT_REGION }}
```
Comment (RobertRostohar) [inline .github/workflows/virtual_hardware.yml:76]: Why is "env" used for AWS_DEFAULT_REGION and "secrets" for AWS_ASSUME_ROLE ?

Comment (spcaipers-arm) [reply .github/workflows/virtual_hardware.yml:76]: Hi Robert, there is no good reason to have `AWS_DEFAULT_REGION` as a secret. Better to use ENV directly.

Comment (RobertRostohar) [reply .github/workflows/virtual_hardware.yml:76]: How about AWS_ASSUME_ROLE? "secret" or "env"?

Comment (spcaipers-arm) [reply .github/workflows/virtual_hardware.yml:76]: I thought about that, just for the sake to play safe, better to have the rest as secrets. 
But all the AWS info in the secrets is low risk IMO.

**Pull request merged**